examples/get-cert.py

```diff
@@ -45,39 +48,34 @@
 api.authenticate(config.API_CLIENT_ID, config.API_CLIENT_SECRET)
 
 #
-# Read the pubkey, strip padding and prefix & suffix
+# Return 2 certificates (SHA2-512 + RSA) in a list
 #
 
 
-def read_pubkey(file):
-    with open(file, "r") as f:
-        _prefix, key_data, *_suffix = f.read().split(" ")
-        return key_data.replace("=", "")
-
-
 def get_cert(target_host_config):
     cert = api.get_target_host_credentials(target_host_config)
     if cert.ok:
         certificates = cert.data.get("certificates")
         if certificates is None or len(certificates) == 0:
-            print("Certificates not found")
+            print(cert.data)
             sys.exit(1)
 
-        return certificates[0]["data_string"]
+        return certificates[0]["data_string"], certificates[1]["data_string"]
     else:
         print(cert.data.get("details"))
         sys.exit()
 
 
 def main():
-    if PUB_KEY_PATH == "" or PUB_KEY_PATH is None:
-        print("public key is not declared")
-        sys.exit(1)
+    # Cleaning the padding
+    conf["public_key"] = conf["public_key"].replace("=", "")
 
-    conf["public_key"] = read_pubkey(PUB_KEY_PATH)
+    # Example
     cert = get_cert(conf)
```

Comment:
Since the signature of the function has changed, the assignment should also be clarified:

```
# Generic extraction...
certList = (...)
# Or, emphasize ordered cert type with variable unpacking
certSHA2, certRSA, *_ = get_cert(...)
```

In addition, replace the 2 prints with a print loop. Try f-string:
```
print(f"{'-'*12} Cert{i}\n{cert[I]}")
```